packages/langium/src/documentation/jsdoc.ts

```diff
@@ -4,11 +4,11 @@
  * terms of the MIT License, which is available in the project root.
  ******************************************************************************/
 
-import type { CstNode } from '../syntax-tree';
+import type { CstNode } from '../syntax-tree.js';
 import { Position, Range } from 'vscode-languageserver';
-import { URI } from 'vscode-uri';
-import { NEWLINE_REGEXP } from '../generator/template-string';
-import { escapeRegExp } from '../utils/regex-util';
+import vscodeUri from 'vscode-uri';
```

Comment:
Not only applicable here, but why did you change the import? Importing and using just `URI` is fine, isn't it?

Reply:
Or is this a problem only at runtime, because `vscode-uri` is cjs package?

Reply:
The problem is at runtime, since `vscode-uri` (even though it is esm) is missing the correct package.json structure. See also https://github.com/microsoft/vscode-uri/pull/25

Reply:
@msujew ok, I guessed there must be a reason. :-) It seems something was merged in June. Let's see when a new version becomes available.